.ci/generate-buildkite-pipeline-premerge

```diff
@@ -147,6 +147,11 @@ function exclude-windows() {
     libc)                ;; # no Windows support
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 126]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Test comment from graphite.dev<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/92694?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->